layout-management-services/src/main/java/org/exoplatform/layoutmanagement/service/LayoutManagementServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package org.exoplatform.layoutmanagement.service;
+
+import org.exoplatform.portal.mop.storage.NavigationStorage;
+
+public class LayoutManagementServiceImpl implements LayoutManagementService {
+
+  private NavigationStorage navigationStorage;
+
+  public LayoutManagementServiceImpl(NavigationStorage navigationStorage) {
+    this.navigationStorage = navigationStorage;
+  }
+
+  @Override
+  public void removeNode(Long nodeId) {
```

Comment:
We need to check if this can be declared and implemented directly into NavigationService layer of gatein-portal